Dante O. Cuales, Jr.

danteocualesjr opened the pull request:
Completed answering function-conversion.js

alexfigliolia commented:
Thank you. Will do!

Alex Figliolia

> On Oct 24, 2017, at 5:45 PM, Emily Sperry <notifications@github.com> wrote:
> 
> Closed #9 <https://github.com/LambdaSchool/JavaScript-II/pull/9>.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/LambdaSchool/JavaScript-II/pull/9#event-1308711431>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ANZESjhQBa3_XUKlR01K6pIU4hR2aaAnks5svlp1gaJpZM4Pv0UW>.
>

danteocualesjr committed:
Completed answering function-conversion.js

danteocualesjr committed:
Updated function-conversion.js

danteocualesjr committed:
Answered questions in array-methods.js

danteocualesjr committed:
Updated array-methods.js

danteocualesjr committed:
Added answer to closure.js

danteocualesjr committed:
Answered Challenge 2

danteocualesjr committed:
Answered Challenge 3

danteocualesjr committed:
Updated array-methods.js

danteocualesjr committed:
Updated function-conversion.js

danteocualesjr committed:
Added answer to Challenge 4

danteocualesjr committed:
Fixed a Bug

There was a bug under Challenge 2 (line 25), so I removed it. Code is now clean/ passed Nodemon.
